src/main/resources/changesets/changelog-consent-48.0.xml

```diff
@@ -5,12 +5,13 @@
     http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet author="grushton" id="48.0-1">
+        <validCheckSum>ANY</validCheckSum>
         <dropAllForeignKeyConstraints baseTableName="user_role_bak"/>
         <addColumn tableName="dacuser">
             <column name="email_preference" type="boolean" defaultValueBoolean="true" />
         </addColumn>
-        <sql>
-            update dacuser set email_preference = false where dacUserId in (select distinct user_id from user_role where email_preference = false)
+        <sql dbms="hsqldb, mysql">
+            update dacuser set email_preference = false where dacuserid in (select distinct user_id from user_role where email_preference = false)
```

Comment:
do you still need dbms="hsql, mysql" here?

Reply:
Yes, to restrict them from failing in postgres.